src/asset_folder_importer/premiere_get_referenced_media/processor.py

```diff
@@ -259,6 +253,9 @@ def process_premiere_project(filepath, raven_client, vs_pathmap=None, db=None, c
                 lg.info("File %s with id %s is already linked to project %s" % (filepath.encode('utf-8'), e.fileid, e.vsprojectid))
             continue
 
+    if invalid_media_paths != []:
+        vsproject.set_metadata({'gnm_project_invalid_media_paths': invalid_media_paths}, mode="add")
```

Comment:
is `vsproject` set here, now you've removed the in-loop load of the project metadata